lib/widgets/content.dart

```diff
@@ -1662,17 +1664,23 @@ class Avatar extends StatelessWidget {
     required this.userId,
     required this.size,
     required this.borderRadius,
+    this.backgroundColor,
+    this.omitPresenceStatus = false,
   });
 
   final int userId;
   final double size;
   final double borderRadius;
+  final Color? backgroundColor;
+  final bool omitPresenceStatus;
```

Comment:
Looking at other call sites of the Avatar constructor, and thinking about whether they should set this flag to omit presence.
* autocomplete → keep
* "My profile" in main menu → probably omit? seems like there's no useful information there; and pending #1618, could be confusing by suggesting you're offline
* lightbox → keep
* msglist senders → see other comment
* new DMs, two places → keep
* top of profile → skip, as this version already does
* users in custom profile fields → keep